File: src/main/java/restaurantmanager/board/Board.java
package restaurantmanager.board;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Entity
@Builder(toBuilder = true)
@AllArgsConstructor(access = AccessLevel.PACKAGE)
@NoArgsConstructor(access = AccessLevel.PUBLIC)
@Getter
@ToString
@EqualsAndHashCode
@Table(name = "boards")
public class Board {
	
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private Long id;
	
	@Column(name = "number_of_seats")
	private Long numberOfSeats;
	
	@Column(name = "board_description")
	private String boardDescription;
	
}

File: src/main/kotlin/restaurant_manager/infrastructure/Board.kt
package restaurant_manager.infrastructure

import javax.persistence.*

@Entity
@Table(name = "boards")
data class Board(

        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        var id: Long? = null,

        var numberOfSeats: Int,

        var description: String,

        @ManyToOne
        @JoinColumn(name = "employee_id", nullable = false)
        var employee: Employee

)
